### PR TITLE
Enforces some good practices in the repository

### PR DESCRIPTION
**Pull request by cjeanner**
The PULL_REQUEST_TEMPLATE.md will be displayed upon PR, in order to ensure contributors are meeting some of the important requirements.

The CONTRIBUTING.md will be shown upon contributor's first PR - and is reachable from the project root in case anyone needs some "how to start" documentation.

### CODE REVIEW

**Comment by cjeanner** [comment]
/retest

galaxy, again.

Maybe we should point to the git repositories directly...

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+$ ansible-galaxy role init --role-skeleton _skeleton_role_ --init-path ci_framework/roles ROLENAME
+```
+
+#### Documentation
+A new role must get proper documentation. Please edit the README.md located in
+the role directory in order to explain its use and details the exposed parameters.
+
+#### Testing
+A new role must get proper Molecule testing. Please edit the default ones, add
+new scenarii if needed.
```
Comment (rlandy) [inline CONTRIBUTING.md:17]: scenario

Comment (cjeanner) [reply CONTRIBUTING.md]: heh, French slipping in the way ;).

**Pull request merged**